examples/ui/ui_texture_slice_flip_and_tile.rs

```diff
@@ -0,0 +1,75 @@
+//! This example illustrates how to how to flip and tile images with 9 Slicing in the UI
```

Comment:
Some links or basic explanation as to what 9-slicing even is would nice for beginners.

Reply:
I think I'd rather make an issue and leave that for a separate PR. The new example is just adapated from the `ui_texture_slice.rs` example which didn't have much in the way of explanation either.